src/main/java/com/labs/introtoprogramming/lab4/image/transform/Scale.java

```diff
@@ -22,13 +22,14 @@ public Scale(double horizontalScale, double verticalScale) {
   public RGBImage applyTo(RGBImage image) {
     int w = image.width();
     int h = image.height();
-    int scaledWidth = (int) (w * horizontalScale);
-    int scaledHeight = (int) (h * verticalScale);
+    int scaledWidth = Math.abs((int) (w * horizontalScale));
+    int scaledHeight = Math.abs((int) (h * verticalScale));
     RGBImage scaledImage = new RGBImage(scaledWidth, scaledHeight);
     double widthRation = (double) (w - 1)
             / (scaledWidth - 1 > 1 ? scaledWidth - 1 : scaledWidth);
```

Comment:
Change here to scalledWidth - 1>= 1 due to bug with scaling picture 2x2 by 1

Reply:
Good catch!